neurons/validator.py

```diff
@@ -843,7 +843,20 @@ async def run_step(self):
                 dim=0,
             )
 
-            # TODO: log here, since the regular wandb log will keep using the original win rates. Or blend the rates.
+            # TODO: Define this competition. This competition ID is used for logging only.
+            # Since we have different win rates for this experimental competition, we need to log it separately.
+            self.log_step(
+                CompetitionId.B7_MODEL_LOWER_EPSILON,
+                uids,
+                uid_to_block,
+                self._get_uids_to_competition_ids(),
```

Comment:
Technically if we use a new competition id here then it won't find any uids since they are all just part of the regular B7 competition.

In this case the step log would show the overall competition id correctly , but then the per uid_data wouldn't be correct.

Perhaps it would be more correct to get this map and manually swap it out for the new competition id before logging? In terms of what would work for the leaderboard?

Reply:
Swap out as in iterate through it and wherever we see 7b comp id, replace with 7b lower epsilon comp id before passing it in.